File: Source/UserManagement/Domain/DataCollector/PhoneNumber/RemovePhoneNumberFromDataCollector.cs
using System;
using System.Collections.Generic;
using Concepts;
using Dolittle.Commands;

namespace Domain.DataCollector.PhoneNumber
{
    public class RemovePhoneNumberFromDataCollector : ICommand
    {
        public DataCollectorId DataCollectorId { get; set; }
        public string PhoneNumber { get; set; }
    }
}

File: Source/UserManagement/Web.Angular/src/app/domain/data-collector/RemovePhoneNumberFromDataCollector.ts
import { Command } from '../../services/Command';

export class RemovePhoneNumberFromDataCollector extends Command {
    dataCollectorId: string;
    phoneNumber: string;

    constructor() {
        super();
        this.type = 'CBS#UserManagement.DataCollector.PhoneNumber-RemovePhoneNumberFromDataCollector+Command@1';
    }
}
